src/config/reader.rs

```diff
@@ -326,7 +330,17 @@ mod test_proto_config {
                 path_to_file_name(path.as_path()).context("It must be able to extract path")?;
             let source = file_rt.read(&path_str).await?;
             let expected = protox_parse::parse(&path_str, &source)?;
-            let actual = helper_map
+            let mut helper_map_clone = helper_map.clone();
+
+            if cfg!(windows) {
+                for item in &mut helper_map_clone {
+                    if let Some(name) = &mut item.name {
+                        *name = name.replace('/', "\\");
+                    }
+                }
+            }
```

Comment:
Let's use a library to do this in a platform independent manner instead of handwriting the logic.